README.rst

```diff
@@ -497,6 +497,15 @@ some system-wide directory like ``/var/zopebackups/instancename/`` and
     a backup from a symlinked directory, in which case
     ``rsync_options = --no-l -k`` does the trick.
 
+``rsync_hard_links_on_first_copy``
+    When using ``rsync`` to make the first backup of the blobs, create hard links
+    instead of copies of the blobs.
+    This is useful to save disk space but you need to pay attention
+    that changing the blobs in the source folder
+    will change the blobs in the backup.
```

Comment:
This should not be the case. Or maybe I misunderstand what you are saying.

How it works in my understanding: when `b.blob` is a hardlink to `a.blob` and you change `a.blob`, then `b.blob` still points to the original file with the original contents. So here: when you change blobs in the source folder, the blobs in the backup do _not_ change.

Now, technically speaking the original blobs never change: they are only added or deleted. But that is an implementation detail of ZODB that we try to ignore in the code. The code should work whether this implementation detail is true or not.

Reply:
To my knowledge after you create an hard link, the "original" and "linked" files are indistinguishable.
They are just human readable labels pointing to an inode.

That's the reason for the caveat.

This is also in the tests:
- https://github.com/collective/collective.recipe.backup/pull/74/files#diff-90f1ad574472ec6881e29e8452aa5c621287919f22e7932769e16e90cc64f667R41-R43
- https://github.com/collective/collective.recipe.backup/pull/74/files#diff-90f1ad574472ec6881e29e8452aa5c621287919f22e7932769e16e90cc64f667R66-R69

That works well enough with the ZODB and allows us to save tons of space:
```shell
$ ./bin/plonebackup
INFO: Please wait while backing up database file: .../filestorage/Data.fs to .../backup/var/fs
INFO: Please wait while backing up blobs from /.../blobstorage to /.../backup/var/blobs
INFO: rsync -a  --link-dest=/.../blobstorage/ /.../blobstorage/ /.../backup/var/blobs/blobstorage.2022-10-04-10-02-06/blobstorage/
INFO: Creating symlink from latest to blobstorage.2022-10-04-10-02-06
$ du -sh /.../blobstorage/ var/blobs/
208G    /.../blobstorage/
6,4G    var/blobs/
```

Reply:
Ah, right. And hard links work fine for us exactly because ZODB never changes a blob file.
Can you add something to that effect in this part of the readme? Now it reads like "if you change the original, then the backup also changes, so you effectively have no backup."